docs/trace_collection.md

```diff
@@ -0,0 +1,95 @@
+# Android Trace Collection
+
+Send [traces][1] to Datadog from your Android applications with [Datadog's `dd-sdk-android` client-side tracing library][2] and leverage the following features:
+
+* Create custom [spans][3] for operations in your application.
+* Add `context` and extra custom attributes to each span sent.
+* Optimized network usage with automatic bulk posts. 
+
+**Note**: Traces are available in the `dd-sdk-android` library version `1.4.0` or higher. The `dd-sdk-android` library supports all Android versions from API level 19 (Kit-Kat).
+
+## Setup
+
+1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
+
+    ```conf
+    repositories {
+        maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+    }
+
+    dependencies {
+        implementation "com.datadoghq:dd-sdk-android:x.x.x"
```

Comment:
Should we mention here about the conflict with the "guava:listenablefuture" module ?

Reply:
Good call !